driver-sync/src/test/functional/com/mongodb/client/unified/Entities.java

```diff
@@ -50,10 +76,12 @@
 import static com.mongodb.client.Fixture.getMongoClientSettingsBuilder;
 import static com.mongodb.client.unified.UnifiedCrudHelper.asReadConcern;
 import static com.mongodb.client.unified.UnifiedCrudHelper.asWriteConcern;
+import static java.util.Collections.synchronizedList;
 import static java.util.Objects.requireNonNull;
 import static org.junit.Assume.assumeTrue;
 
-final class Entities {
+public final class Entities {
```

Comment:
This class has to be public now so it can be accessed by the workload executor to write the results files.